lib/util/http/github.ts

```diff
@@ -223,14 +223,18 @@ export class GithubHttp extends Http<GithubHttpOptions, GithubHttpOptions> {
     return result;
   }
 
-  private async getGraphql<T = unknown>(
+  public async queryRepo<T = unknown>(
     query: string,
-    accept = 'application/vnd.github.merge-info-preview+json'
-  ): Promise<GithubGraphqlResponse<T>> {
+    options: GraphqlOptions = {}
+  ): Promise<T> {
     let result = null;
 
     const path = 'graphql';
 
+    const {
+      acceptHeader: accept = 'application/vnd.github.merge-info-preview+json',
```

Comment:
Isn't it a mistake to have it in this file? Planning to correct it in a follow-up PR?

Reply:
Right, after `initRepo` refactoring PR

Reply:
If it requires a fix or change to how headers are handled in this graphql, can it be done before?